$jqueryEl.val() now puts spaces between value and unit

kylelamse opened the pull request:
Before when you did `$('#duration-picker').val()` you would get something like `0h, 0m, 0s`.  It makes more sense to have something like `0 h, 0 m, 0 s`.

Tartarus762 commented:
Why does it make more sense to have spaces? It should not make much difference; a string can be split up into characters. I assume you want to use `$('#duration-picker').val().split(" ");` or similar? If you're using Javascript, for example, you can just use `$('#duration-picker').val().split("");` which is the same as above but without the space, you can also just access the characters in a string as though it were an array: `$('#duration-picker').val()[0]` would be 0,` [1]` would be "h" etc.

I don't mean to sound rude, I would just like you to explain to me exactly why this would improve the plugin.

kylelamse commented:
Because it is far simpler and more predictable imo.
```javascript
$('#duration-picker').durationPicker({
  months: {
    min: 0
    max: 12
  },
  weeks: {
    min: 0
    max: 4
  },
  days: {
    min: 0
    max: 31
  }
});
```
All I would have to do is
```javascript
$('#duration-picker').val().split(',').map(chunk => chunk.split(' '));
```

I could change a key from `months` to `supercalifragilisticexpialidocious` for example and my above call would still work.  It makes sense to me, but in the end it's your project so your call.  Great work on this though, I appreciate this plugin you made.

Tartarus762 commented:
That is actually a good point, as it stands, it only works easily for single character keys. 

I'll merge the request soon, thanks for the input.
